src/common/underline.js

```diff
@@ -0,0 +1,13 @@
+const underline = (e) => {
+  if (e.currentTarget.checked === true) {
+    const index = e.currentTarget.parentNode.childNodes[0].innerHTML;
+    console.log(index);
+    e.currentTarget.parentNode.childNodes[3].style.textDecoration = 'line-through';
+    e.currentTarget.parentNode.childNodes[3].style.color = 'gray';
+  } else {
+    e.currentTarget.parentNode.childNodes[3].style.textDecoration = 'none';
+    e.currentTarget.parentNode.childNodes[3].style.color = 'black';
+  }
+};
+/* eslint-disable */
+export { underline }
```

Comment:
- [ ] As I mentioned in the previous comment(s), Kindly make sure you fix all your `linters` errors instead of disabling them as you did for best practices and clean code.

Reply:
Changed the same.